### DISCUSSION TITLE
Tax free after holding assets for 1 year

### DISCUSSION THREAD

**Discussion by RichieGKB** [provinzio/CoinTaxman#160] [Q&A] [answered]
Does the code contain, that profits from assets are tax free if they have been hold for at least 1 year?
This is currently given by the german tax law.
But I am not able to identify the corresponding place in the code and want to prevent declaring more relevant profit than potentially occured.
Did someone already figure that out?

**Comment by provinzio** [comment, marked as answer]
Sells 1 year after the buy aren't counted as taxable in the tool. 

https://github.com/provinzio/CoinTaxman/blob/10675553cc23e7d28f99813e1e92cb9b1a7f17d5/src/taxman.py#L280
